crates/astria-conductor/src/block_cache.rs

```diff
@@ -49,7 +49,10 @@ impl<T> BlockCache<T> {
     /// Returns the next sequential block if it exists in the cache.
     pub(crate) fn pop(&mut self) -> Option<T> {
         let block = self.inner.remove(&self.next_height)?;
-        self.next_height += 1;
+        self.next_height = self
+            .next_height
+            .checked_add(1)
+            .expect("block height should not be `u64::MAX`");
```

Comment:
```suggestion
            .expect("block height must not exceed `u64::MAX`");
```

Reply:
Done in [44c99af](https://github.com/astriaorg/astria/pull/1081/commits/44c99affc52d66ee3134fa1bbc23f66d21bb632a).